Configuration/Settings.yaml

```diff
@@ -8,4 +8,9 @@ Neos:
     mvc:
       routes:
         'Sitegeist.Kaleidoscope':
-          position: 'before Neos.Neos'
\ No newline at end of file
+          position: 'before Neos.Neos'
+
+Sitegeist:
+  Kaleidoscope:
+    fallbackFormat: 'png'
+    overrideImagineDriver: false
```

Comment:
I suggest to move those two options below a key "dummyImage" as they do nor affect other images.

Reply:
Perfect, will do so.

Reply:
Done